### PR TITLE
Fix bug where SparkConfigMap is ignored when re-loading from SparkSubmissionParameter

### PR DESCRIPTION
**Pull request by djlan**
Fix bug where SparkConfigMap is ignored when re-loading from SparkSubmissionParameter

### CODE REVIEW

**Comment by vscjavaci** [comment]
Can one of the admins verify this patch?

**Comment by djlan** [comment]
retest this please

**Comment by djlan** [comment]
Close pull request to re-trigger build

**Comment by vscjavaci** [comment]
Can one of the admins verify this patch?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java [hunk 1]
```diff
@@ -534,6 +534,27 @@ private void initializeTableModel(final InteractiveTableModel tableModel) {
...
                         configs.get(SparkSubmissionParameter.parameterList[i]) : "");
             }
+
+            for (Map.Entry<String, Object> jobConfigEntry : configs.entrySet()) {
+                String jobConfigKey = jobConfigEntry.getKey();
+                Object jobConfigValue = jobConfigEntry.getValue();
+
+                if (!StringHelper.isNullOrWhiteSpace(jobConfigKey) && !SparkSubmissionParameter.isSubmissionParameter(jobConfigKey)) {
+                    if (jobConfigKey == SparkSubmissionParameter.Conf) {
+                        Map<String, Object> sparkConfigMap;
```
Comment (wezhang) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java]: use `SparkConfigures` here

Comment (djlan) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java]: Updated. Will send out a new PR

---
In reply to: [117957965](https://github.com/Microsoft/azure-tools-for-java/pull/542#discussion_r117957965) [](ancestors = 117957965)

**Comment by djlan** [comment]
retest this please

**Pull request merged**